clang/include/clang/Basic/TargetOptions.h

```diff
@@ -85,6 +85,9 @@ class TargetOptions {
   /// backend
   bool SYCLUseVXXNames : 1;
 
+  /// \brief generate code targeting the version2 of the acap/aie lib
```

Comment:
I guess you could add another commit to remove the old lib support.

Reply:
when talking to you I understood we should keep support for the old library.